source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,129 @@
+::
+	server:
+            # the interface that is used to connect to the network, this means on this machine
+            interface: 0.0.0.0
+            interface: ::0
+            # addresses from the IP range that are allowed to connect to the resolver
+            access-control: 10.0.0.0/8 allow
+            access-control: 2001:DB8::/64 allow
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the 10.0.0.0/8 IP range.
+
+To prepare our config we are going to modify the existing config in /etc/unbound/unbound.conf. 
+If you open the file we see that there is already an “include” in there. This include enables us to do DNSSEC, which allows Unbound to verify the source of the answers that it receives [LINK ?], so we want to keep this. If you don't have the files they can be created using the *unbound-anchor* command.
+
+With your favourite text editor then add the minimal config as shown above, making any changes to the access control where needed. Do note that we strongly recommend to keep the *include* that is already in the file. When you are happy with our config, we first need to kill the currently running unbound server and restart it with our new configuration.
+
+you can kill the current version with 
+::
+	pkill -f unbound
+
+And you can restart Unbound with:
+::
+	unbound -c /etc/unbound.conf
+
+Testing the resolver from a remote machine
+------------------------------------------
+
+So now we have a DNS resolver which should be reachable from within the network. To verify this we need to find the IP address of the resolver machine which can be found on the machine itself. For this tutorial we will use the address “10.10.10.10” (not 127.0.0.1 as we saw earlier) as an example. Armed with the IP address we can send a query to our DNS resolver from another machine which is within our home network. To do this we use the same dig command, only we change the IP address where the query is asked.
+::
+	dig example.com @10.10.10.10
+
+This should give the same result as the query from the local test.
+
+
+Where it all comes together
+---------------------------
+
+We should now have a functioning DNS resolver that is accessible to all machines in our network. 
+
+The next step then becomes a little tricky. We have a choice of which machines in our network will be using our configured DNS resolver. This can range from a single machine to all the machines that are connected. 
+
+Since this tutorial cannot (and does not try to) be comprehensive, we wil look at some of the basic examples on which you can expand.
+
+While not all, some machines use the resolver “recommended” by your router. To change this, we need to log into the router and configure it to use the DNS resolver that we just set up. This configuration step varies greatly from vendor to vendor, but the rule of thumb is that your router is accessible on either 192.168.1.1 or 192.168.0.1.
```

Comment:
Since this is a tutorial for those comfortable with a terminal, perhaps suggest to look at the routing table to find the default gateway and thus the router IP if not already known?

Reply:
Good idea!